### PR TITLE
PLATY-410: DownloadController integration tests

### PR DESCRIPTION
**Pull request by JoPintoPaul**

### CODE REVIEW

File: it/controllers/DownloadControllerISpec.scala [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  implicit val actorSystem: ActorSystem        = ActorSystem()
+  implicit val materializer: ActorMaterializer = ActorMaterializer()
+  implicit val timeout: akka.util.Timeout      = 10.seconds
+
+  "DownloadController" should {
+
+    "download a file" in {
+      Given("a reference to a previously stored file")
+
+      val path = Paths.get("it/resources/my-it-file.txt")
```
Comment (tomasz-rosiek) [inline it/controllers/DownloadControllerISpec.scala]: This writes to 'it' directory. There is a risk that someone accidentially commit this file.
Should we use something like temporary directory?

Comment (JoPintoPaul) [reply it/controllers/DownloadControllerISpec.scala]: @tomasz-rosiek If the code works successfully, then the file is "moved" to the /tmp directory by the FileStorageService, which removes it from the it/resources directory as part of the test. So when the test runs as expected, this file will be in those location only for a few milliseconds before being moved to /tmp by the FileStorageService.

Having said that, if the FileStorageService code is broken, then this file could be not moved correctly. Maybe seeing the file appearing would be a sign that the test is broken? In fact, I could add that check in during the test (no longer in original location). What do you think?

Comment (cybersaurus) [reply it/controllers/DownloadControllerISpec.scala]: java.io.File.createTempFile(...) ?

Comment (JoPintoPaul) [reply it/controllers/DownloadControllerISpec.scala]: @cybersaurus Made suggested correction to code.

**Pull request merged**